doc/scheduler-job-examples.tex

```diff
@@ -1,7 +1,7 @@
 % ------------------------------------------------------------------------------
 \subsection{Example Job Script: Fluent}
 
-\begin{figure}[htpb]
+\begin{figure}[h!]
```

Comment:
Why this change? htpb is generally good and sufficient for this.

Reply:
fixed